data/style.css

```
@@ -29,7 +29,8 @@ footer {
 }
 
 img {
-	max-width: 600px ;
+	padding-left: 120px;
+	max-width: 250px ;
```

```suggestion
	max-width: 50%;
```
120px shows up tiny on my monitor. I think 50% of page is probably what you want, but I'm not sure if other people think the same.

---

I'm not sure about the exact value, but I agree that having the width value be a percentage instead of an absolute pixel value does make a lot of sense. If we wanted to have the maximum value be an absolute pixel width, we should enforce that as a condition for submitting images instead of in the CSS.

---

MMmmmh if you wanted why not using the resizer tool chrome provide we ahd it in windows-xp

---

The only issue with using a percentage is that mobile will have issues. 
Personally, I think we should go with a percentage-based max width (or maybe even just width) for images, and then a media rule to expand it in portrait orientations such as:
`
img {
    max-width: 60%;
}
@media screen and (orientation:portrait) {
    img {
        max-width: 85%;
    }
}`
with screen specified so we don't mess with printers.